### ISSUE TITLE
Names for Arm64 system registers need to be updated

### ISSUE THREAD

**Issue by plafosse** [Vector35/binaryninja-api#4176]
```
Unknown system register 55303 @ 0x12a48: S3_3_c0_c0_7, using generic system register instead
Unknown system register 57089 @ 0xf0ec: S3_3_c14_c0_1, using generic system register instead
Unknown system register 55297 @ 0xa980: S3_3_c0_c0_1, using generic system register instead
```

According to this:  they correspond to: https://gist.github.com/bazad/42054285391c6e0dcd0ede4b5f969ad2
```
    'S3_3_c0_c0_7'   : ( 'DCZID_EL0', 'Data Cache Zero ID register' ),
    'S3_3_c14_c0_1'  : ( 'CNTPCT_EL0', 'Counter-timer Physical Count register' ),
    'S3_3_c0_c0_1'   : ( 'CTR_EL0', 'Cache Type Register' ),
```

**Comment by negasora** [comment]
Fixed in `4.1.5464-dev`

**Comment by galenbwill** [comment]
But wait... there's more!
```
[0:0 Default warn] Unknown system register 55497 @ 0x0: S3_3_c1_c9_1, using generic system register instead
[0:0 Default warn] Unknown system register 62015 @ 0x0: S3_6_c4_c7_7, using generic system register instead
[0:0 Default warn] Unknown system register 44457 @ 0x0: S2_5_c11_c5_1, using generic system register instead
[0:0 Default warn] Unknown system register 36348 @ 0x0: S2_1_c11_c15_4, using generic system register instead
[0:0 Default warn] Unknown system register 51865 @ 0x0: S3_1_c5_c3_1, using generic system register instead
[0:0 Default warn] Unknown system register 58522 @ 0x0: S3_4_c9_c3_2, using generic system register instead
[0:0 Default warn] Unknown system register 41073 @ 0x0: S2_4_c0_c14_1, using generic system register instead
[0:0 Default warn] Unknown system register 40049 @ 0x0: S2_3_c8_c14_1, using generic system register instead
```
